aws/requests.yaml

```diff
@@ -16,6 +16,11 @@ releases:
         - name: coredns
           version: ">= 1.4.0"
           issue: https://github.com/giantswarm/giantswarm/issues/13771
+    - name: "> 14.1.1"
```

Comment:
WDYT about including it in 15.0.0 instead?

Reply:
For AWS and Azure this is actually only a bump of 2 release versions, so not as big a jump as it looks. I'm also fine with 15, just trying to keep new stuff as fresh as possible and cut down the tail a little bit

Reply:
I am just concerned if we will get pinged if something goes wrong and teams are only trying to release patches as everyone is focussed on CAPI.

Reply:
They can always add an exception if it blows up, no harm done